src/main/java/seedu/address/logic/LogicToDo.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic;
+
+import javafx.collections.ObservableList;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.todolist.ToDoListEvent;
+
+/**
+ * API of the LogicToDo component
+ */
+public interface LogicToDo {
+    /**
+     * Executes the command and returns the result.
+     *
+     * @param commandText The command as entered by the user.
+     * @return the result of the command execution.
+     * @throws CommandException If an error occurs during command execution.
+     * @throws ParseException   If an error occurs during parsing.
+     */
+    CommandResult execute(String commandText) throws CommandException, ParseException;
```

Comment:
we shouldn't have two logic interfaces. I think that the Logic should serve as the facade for both the calendar and the todolist in the model

Reply:
Agree with Bryan, we can make this implement the Logic interface. We can just add a new method to get the ObservableList of the ToDoList items.